src/select.coffee

```diff
@@ -492,8 +531,10 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
 
   actions:
     toggleDropdown: (event) ->
-      return if @get('disabled')
-      @toggleProperty 'showDropdown'
+      if @get 'showDropdown'
+        @send 'hideDropdown' if not @get('lockDropdownOnShow')
```

Comment:
Why handle this differently than other cases of showDropdown/lockDropdownOnShow?